pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/migration/KitRequestShippingMigrator.java

```diff
@@ -14,7 +14,7 @@ public KitRequestShippingMigrator(String index, String realm) {
 
     @Override
     protected Map<String, Object> getDataByRealm() {
```

Comment:
Might be worth a method comment so that the caller knows the type of the values in the map.